src/uu/cp/src/platform/windows.rs

```diff
@@ -0,0 +1,159 @@
+use std::{
```

Comment:
please add the copyright header
```suggestion
//  * This file is part of the uutils coreutils package.
//  *
//  * For the full copyright and license information, please view the LICENSE
//  * file that was distributed with this source code.

use std::{
```

Reply:
Should I additionally add my name someplace as it looks like there's no CLA involved? Also: whoops! Thanks for the reminder

Reply:
If you want you can put your name on top of the file but you don't have to. There's indeed no CLA involved. You keep the copyright of your code, but license it as MIT by contributing.